src/handlers/domanda.py

```diff
@@ -17,7 +17,8 @@ def domanda(update: Update, context: CallbackContext) -> None:
         keyboard.append([button])
 
     reply_markup = InlineKeyboardMarkup(keyboard)
-    context.bot.send_message(chat_id=update.effective_chat.id, text=testo, reply_markup=reply_markup)
+    rm_message = context.bot.send_message(chat_id=update.effective_chat.id, text=testo, reply_markup=reply_markup)
+    context.job_queue.run_once(fine_timer,10,context=(update,context,rm_message.message_id))
```

Comment:
Qui si decide la durata del timer. Essa dovrebbe variare in base alla difficoltà scelta dall'utente